src/test/cypress/fixtures/requests/programming_exercise_exam_template.json

```diff
@@ -0,0 +1,44 @@
+    "title": "Default cypress programming exercise name",
+    "shortName": "DefaultShortName",
+    "packageName": "default.package.name",
+    "maxPoints": 10
+}
```

Comment:
As you already mentioned in my PR this template is redundant and can be deleted

Reply:
How does it keep popping back up? :sweat_smile:

Reply:
Done